inits.json

```diff
@@ -0,0 +1,54 @@
+{
+    "user_info": [
+        {
+            "Directory with FITS files": "./sample-data/HatP32Dec202017",
+            "Directory to Save Plots": "./sample-data",
+            "Directory of Flats": "n",
+            "Directory of Darks": "n",
+            "Directory of Biases": "n",
+
+            "AAVSO Output?": "y",
+            "AAVSO Observer Account Number": "RTZ",
+            "Secondary Observer Codes": "n",
+
+            "Observation date": "December 17, 2017",
+            "Obs. Latitude (+=N,-=S)": "+31.68",
+            "Obs. Longitude (+=E,-=W)": "-110.88",
+            "Obs. Elevation (meters)": 2616,
+
+            "Camera Type (CCD or DSLR)": "CCD",
+            "Pixel Binning": "1x1",
+            "Exposure Time (seconds)": "60",
+            "Filter Name (aavso.org/filters)": "V",
+            "Observing Notes": "Weather, seeing was nice.",
+
+            "Target Star X & Y Pixel": [424, 286],
+            "Comparison Star(s) X & Y Pixel": [[465, 183], [512, 263]]
+        }
+    ],
+    "planetary_parameters": [
+        {
+            "Planet's Name": "HAT-P-32 b",
```

Comment:
should we give the user some guidance here that their star name must match how it is listed in the NASA Exoplanet Archive?

Reply:
If the exoplanet is in the NEA, it already scrapes the planet and star names from the NEA and ignores the inits.json inputs. The values when using an exoplanet in NEA that EXOTIC automatically uses are the planet name, star name, right ascension, and declination. All other parameters will be taken from inits.json. If this is a candidate planet, it will use all the inits.json parameters.

Reply:
RA and DEC values are now matched to the NEA for comparison and the user can choose between the two or enter a new one.